.tools/bin/update-root-composer

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+$replace = [];
+
+$baseDir = dirname(__DIR__, 2);
+$dirs = [
+    $baseDir.'/redaxo/src/addons/be_style',
```

Comment:
vllt besser

```suggestion
    glob($baseDir.'/redaxo/src/addons/*/composer.lock');
```
?

Reply:
Dann würde man auch Fremdaddons erwischen (man führt das ja lokal aus, und ich habe lokal immer auch Fremdaddons in der Installation liegen)

Reply:
Ok, dann am besten nen kommentar wann hier addons zu listen sind